projects/common/src/utilities/validators/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './domain-validator';
```

Comment:
This change looks fine to me. I do not think we use `index.ts` file anywhere as of now.
IMO, I think we can use this pattern.

cc: @aaron-steinfeld @anandtiwary @arjunlalb

Reply:
I don't have a problem with it